src/com/google/enterprise/adaptor/database/UniqueKey.java

```diff
@@ -217,8 +224,8 @@ static String decodeSlashInData(String id) {
 
   @Override
   public String toString() {
-    return "UniqueKey(" + docIdSqlCols + "," + types + "," + contentSqlCols + ","
-        + aclSqlCols + ")";
+    return "UniqueKey(" + docIdSqlCols + "," + types + ","
+        + contentSqlCols + "," + aclSqlCols + "," + metadataSqlCols + ")";
```

Comment:
Why re-wrap this and modify the first line? Random or on purpose?

Reply:
The line was triggering my 80-cols check and since I was adding to the string anyway, I changed it to get rid of the warning.